src/models/selection.ts

```diff
@@ -3,4 +3,5 @@ import { RunGraphNode } from '@/models/RunGraph'
 export type NodeSelection = {
   id: string,
   kind: RunGraphNode['kind'],
+  artifacts?: RunGraphNode['artifacts'],
```

Comment:
Why do we need to emit all the artifacts?

Reply:
It's actually being emitted anyway, since the whole node is emitted. This is just a collection of properties that are consistently emitted between flow and task selections. Including it in the type here means that when I call `getSelection` I can check if the inspected artifact is associated with the selected node.